src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,905 @@
+
+As Ron’s cart crested a hill, it clipped a large rock. I saw the cart rise and flip beautifully like an angel underwater. If I could have one image to ruminate on for the rest of my life, I would choose this one.
+
+[[I headed off into the night|The Gutter - Ending hub]] towards a future that could only disappoint me in comparison.
+
+::The Gutter - Ron's car finale - No key
+When he realized he didn’t have the key, he panicked. He searched all over himself and then all over the car. But, if he had a spare, it wasn’t where it should be.
+
+Out of options, Ron tried to kick one of the wheels of the car but got his foot caught in the barbed wire instead. He struggled for a bit, but eventually gave up and fell asleep in the grass. After a few minutes passed, I snuck down and wrote “Fart Angel” on his forehead.
+
+[[I headed off into the night|The Gutter - Ending hub]] towards a future that could only disappoint me in comparison.
+
+::The Gutter - Anna's letter
+The letter was thick with Ron’s beer sweat stench, but I didn’t care. As soon as I saw her handwriting my eyes welled up and my fists clenched.
+
+(click: "saw her handwriting")[<div style="padding-left: 2em; font-style: italic;">[$dissolve[$playerName,
+
+God damn sis. I’m so fucking sorry. If you’re feeling how I think you’re feeling right now, just know that I feel the same or worse. That shit I bought from Dan should wear off in a few hours.
+
+I’m the worst kind of backstabber. Trust that Judas never looked this fresh though. ;)
+
+I just didn’t know what else to do. I realized after we talked that we didn’t want the the same things. That’s OK. I still love you. But, I’ve gotta leave.]]
+
+(click: "gotta leave")[$dissolve[I can’t take another beating or groping or whistle or side-eyed glance or ass pat or hair pull or cat call. I can’t eat another Twinkie and call it dinner because the meal coins got messed up again. I can’t spend another week scrapping for the book I want, only to find it torn up and covered in dick drawings.
+
+Please understand. I hope you understand. I hate this place more than any other place on Earth. I hope it burns down. I’d press the button to nuke it from orbit. I mean, if you weren’t there.
+
+Fuck. Why is it so hard to leave? We spent all those nights raging, and stirring ourselves to howling, and dreaming of a better life, so why is it so hard to find the energy to get out of this vortex of shit?]]
+
+(click: "vortex of shit")[$dissolve[They had us playing a rigged game the whole time. You know the mountain? Their fucking stupid path to an imaginary top. I found a different way through and I’m peacing outta this bitch. I hope they think I died up there, another stupid bitch crawling for the crown.
+
+You know what, I might die and I don’t even care. In the end, it’s my risk and I cou
```

Comment:
Might be another good place for the player to do some character definition here. Like two options:

> `>>` It took me a while to stop sobbing—
> `>>` (something something else, if the player is trying to play the hardass type)—

Reply:
Oh yeah, that's an interesting option. I like it! Maybe something where the player stops themselves from crying like:

"I clenched my jaw tight and dug my fingernails into my arm until the threat of tears passed."